app/Http/Controllers/Accessories/AccessoriesController.php

```diff
@@ -115,6 +115,33 @@ public function edit($accessoryId = null)
 
     }
 
+    /**
+     * Returns a view that presents a form to clone an accessory.
+     *
+     * @author [J. Vinsmoke]
+     * @param int $accessoryId
+     * @since [v6.0]
+     * @return View
+     */
+    public function getClone($accessoryId = null)
+    {
+
+        $this->authorize('create', Accesory::class);
+
+        // Check if the asset exists
+        if (is_null($accessory_to_clone = Accessory::find($accessoryId))) {
+            // Redirect to the asset management page
+            return redirect()->route('accessory.index')->with('error', trans('admin/accessories/message.does_not_exist'));
```

Comment:
Routes are usually plural, so this should have been `accessories.index`, versus `accessory.index`. Fixed in #13439